plugins/cpp_metrics/model/include/model/cpprelationalcohesion.h

```diff
@@ -0,0 +1,117 @@
+#ifndef CC_MODEL_CPPRELATIONALCOHESION_H
+#define CC_MODEL_CPPRELATIONALCOHESION_H
+
+#include <model/cppentity.h>
+#include <model/cpprecord.h>
+#include <model/cppastnode.h>
+#include <model/cppfunction.h>
+#include <model/cppvariable.h>
+
+namespace cc
+{
+    namespace model
+    {
+
+        #pragma db view \
+        object(File)
+        struct RelationalCohesionFileView
+        {
+            #pragma db column(File::path)
+            std::string filePath;
+
+            #pragma db column(File::type)
+            std::string fileType;
+
+            #pragma db column(File::id)
+            std::size_t fileId;
+
+        };
+
+        #pragma db view \
+        object(CppRecord) \
+        object(CppAstNode : CppRecord::astNodeId == CppAstNode::id) \
+        object(File : CppAstNode::location.file) \
+        object(CppMemberType : CppMemberType::memberAstNode) 
+        struct RelationalCohesionRecordView
```

Comment:
I think the same view is used in #757. In this case a more generic name should be used and code duplication should be avoided.

Reply:
A common view could probably be used however the same applies to "CohesionCppRecordView" for example ("RelationalCohesionRecordView" contains this + typeHash). 

Maybe this should be looked at separately to unify the commonly used views?

Reply:
You started working concurrently on a second issue before finishing this one. I don't advise to merge in duplicated code segments so close to each other in time, and then refactor them in a follow-up issue.